src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -198,6 +200,49 @@ def endAuction():
 
 This example should give you a sense of what Vyper contract syntax is like. For a more detailed description of the functions and variables, [see the docs](https://vyper.readthedocs.io/en/latest/vyper-by-example.html#simple-open-auction).
 
+## Yul and Yul+ {#yul}
+
```

Comment:
I think it might be nice to have a short introduction to these two languages. I realise that there's an intro right at the top of the page but I feel like a lot of people would miss this.

Reply:
Yeah but then we basically need to repeat the same all over again --> Yul is an intermediate language for Ethereum. There's not much more to say than the stuff that is mentioned in the bullet points below.

Reply:
Why would someone want to use Yul or Yul+ ? After a Google, I came across this description of Yul+ "It allows the developer to get much closer to raw EVM than Solidity, and with that comes the promise to drastically improve gas usage."

Could we say something like "Yul gives you more control for optimising your smart contract, like reducing the amount of Gas it needs and..."

What are the other outcomes a developer might be able to achieve by using Yul instead of Solidity – hope this makes sense.

Reply:
It does make sense but then we're repeating what is said in the comparison section. :) There is also no such "arguments" in the Vyper or Solidity sections (just to be consistent).

Reply:
Also, the "Intro to Yul+" post that you googled is already linked below in the important resources section. :)

Reply:
I was trying to follow the structure of the page and put advantages of the language in the comparison section and the objective traits of the lang in the introductory bullet points, just like it's being done with the other two languages.